datadog_lambda/tracing.py

```diff
@@ -220,7 +220,11 @@ def extract_context_from_sqs_or_sns_event_or_context(event, lambda_context):
     try:
         context = _extract_context_from_eventbridge_sqs_event(event)
         if _is_context_complete(context):
-            return context
+            return (
+                context,
+                # Unsupported DSM context propagation method
+                None,
+            )
```

Comment:
What happened to the code we worked on together? How we created a new method to call instead of calling `propagator.extract`? That is the proper way to do this. We should not be changing the signature of all of these methods.